tests/governance_js.py

```diff
@@ -507,7 +507,7 @@ def test_apply(network, args):
             "/gov/proposals",
             proposal(action("always_throw_in_apply")),
         )
-        assert r.status_code == 200, r.body.text()
+        assert r.status_code == 500, r.body.text()
```

Comment:
Could we add a check that bundles 2+ actions together and check that if the last one fails, the proposal is still marked as `Failed`? I observed this with the bundling of the service open proposal.

Reply:
Absolutely!

Reply:
I did, it didn't work, and I scratched my head for a bit trying to work out why. To save anyone else doing the same: the test constitution only applies the semantic that the proposal names suggest when they are by themselves, and defaults to voting otherwise.